servers/quarkus-server/src/testFixtures/java/org/projectnessie/server/AbstractQuarkusRest.java

```diff
@@ -28,7 +28,7 @@
 @ExtendWith(QuarkusNessieClientResolver.class)
 public abstract class AbstractQuarkusRest extends BaseTestNessieRest {
 
-  @Inject Instance<Persist> persist;
+  @Inject public Instance<Persist> persist;
```

Comment:
Do we have to make fields `public` too? The Quarkus issue appears to be related to class loading :thinking:

Reply:
Only in this case. And it's rather a workaround for now. But it's test code - so it doesn't really hurt.